book-2-platform-api/chapters/API_MODELS_CONTROLLERS.md

```diff
@@ -218,7 +357,7 @@ Your instruction team will get you started on converting your student exercises
         },
         "AllowedHosts": "*",
         "ConnectionStrings": {
-            "DefaultConnection": "Server=YourSQLServerName\\SQLEXPRESS;Database=StudentExercises;Trusted_Connection=True;"
+            "DefaultConnection": "Server={{Insert your server name here}}\\SQLEXPRESS;Database=StudentExercises;Trusted_Connection=True;"
```

Comment:
Should they not all be running it with `localhost`?  should be able to standardize on either `.`, `(local)` or `localhost`.

Reply:
Didn't even know that `localhost` worked here. I'll update.